src/odemis/dataio/csv.py

```diff
@@ -132,6 +132,26 @@ def export(filename, data):
             csv_writer.writerow(headers)
             csv_writer.writerows(rows)
 
+    elif data.metadata.get(model.MD_ACQ_TYPE, None) == model.MD_AT_EK:
+        logging.debug("Exporting angular spectrum data to CSV")
+
+        spectrum_range, unit_c = spectrum.get_spectrum_range(data)
+        if unit_c == "m":
+            spectrum_range = [s * 1e9 for s in spectrum_range]
+            unit_c = "nm"
+
+        angle_range, _ = spectrum.get_angle_range(data)
+        unit_a = "°"
```

Comment:
Similarly as above, add a `if unit_a == "rad": `... 
Because if it's px, it's not a good idea to convert to °.